src/services/graphql/errorHandler.js

```diff
@@ -1,4 +1,4 @@
-export const errorHandler = (error, history) => {
+export default (error, history) => {
```

Comment:
Let's talk about this one too - way may need to update this (or consuming code).

Reply:
Would also like to discuss this. I think we should think about a different paradigm concerning this, I see a lot of duplication

Reply:
Agreed.  Error handling needs some love.